Apps/WebClient/src/ClientApp/src/views/reports.vue

```diff
@@ -88,6 +93,12 @@ export default class ReportsView extends Vue {
                 text: "Immunizations",
             });
         }
+        if (this.config.modules["MedicationRequest"]) {
```

Comment:
we should be able to use constants here: EntryType.MedicationRequest

Reply:
true, we can address that for the other reports as well on the refactor!